### ISSUE TITLE
Move ability improvements from species to Class and/or Background

### ISSUE THREAD

**Issue by mlenser** [mlenser/kryx-rpg-issues#413]

**Comment by Paulorpribeiro** [comment]
I know you didn't ask for any input to push this, but can I ask, what motivated this change?

**Comment by mlenser** [comment]
> I know it’s part of the dregs of older editions that has endured, I myself would prefer class-based modifiers as well (and hope any future editions lean this way).

https://twitter.com/matthewmercer/status/1243249526846058496

_____________________

There are tons of blogs, tweets, and other opinions on why. There are inclusivity reasons, diversifying character options reasons, and many others. Some quick googles and skimming:

- Inclusivity: https://www.belloflostsouls.net/2020/03/dd-moving-past-racial-ability-scores-using-class.html
- Diversity of choice: https://www.reddit.com/r/dndnext/comments/cwt5qy/classbased_attribute_bonuses_a_way_to_diversify/

**Comment by Paulorpribeiro** [comment]
Ok, so inclusivity and diversity? I know these are the trends today, but what if I want stereotypical in my world? When we see a supplement about a new setting, the first thing they change mechanically is the races, their bonuses and traits. Each world has its own characteristics to races. If you put these traits and bonuses into the classes, it is way harder to use the Krix set of rules with the different settings. If a setting has half orcs that are more prone to spell casting and mental skills rather than brute force, how do you adapt them into this system?

I think a better way to achieve what you are looking for is splitting the difference. I'm gonna talk about ability scores, but the same can be done with abilities, I guess

Regular species have 2 to one ability score and one to the other. I think we could put the +2 into the class and the +1 into the species, for example. You still have a strong characteristic of the species that you can also adapt for each setting, but the class bumps it's main ability score, so every species can still be effective in any class.

**Comment by itamarcu** [comment]
> what if I want stereotypical in my world?

You can always make all of your elves play as rangers and all the half-orcs play as barbarians. The new rules won't stop you from playing to the stereotype, they'll just no longer prevent you from playing against it.

**Comment by mlenser** [comment]
Sharing what I wrote on discord as not everyone uses it:

Even if you wish to have prejudices built into your game (which.. is weird), the methodology is flawed. Saying ***all*** members of a species have a certain ability benefits would simply not be accurate. Forcing a general cultural trend on ***all*** members of species does not reflect the diversity that exists in species.

Additionally, there are tons of blogs and topics about the problematic side of using the term race and the stereotypes that you're talking about. Some quite famous people.

Mercer: `I know it’s part of the dregs of older editions that has endured, I myself would prefer class-based modifiers as well (and hope any future editions lean this way).`
Author N.K. Jemisin: `Orcs are human beings who can be slaughtered without conscience or apology…. Creatures that look like people, but aren’t really. Kinda-sorta-people, who aren’t worthy of even the most basic moral considerations, like the right to exist. Only way to deal with them is to control them utterly a la slavery, or wipe them all out. Huh. Sounds familiar…. The whole concept of orcs is irredeemable. Orcs are fruit of the poison vine that is human fear of ‘the Other.’ In games like Dungeons & Dragons, orcs are a ‘fun’ way to bring faceless savage dark hordes into a fantasy setting and then gleefully go genocidal on them…. They’re an amalgamation of stereotypes. And to me, that’s no fun at all.`

James Mendez Hodes
`…racist myths from the British academy and army fueled J.R.R. Tolkien’s creation of orcs as an analogue for Asian people…. D&D, like Tolkien, makes race literally real in-game by applying immutable modifiers to character ability scores, skills, and other characteristics. The in-game fiction justifies these character traits as absolute realities; they also just happen to be the same cruel and untrue things racists say about different ethnicities, which I am frequently told is a coincidence or makes sense in the game or something.`

`“Where [racism] flashes up in its most obvious forms are in how the game instructs players and DMs to treat the ‘half’ races (specifically half-elves and half-orcs), how POC are depicted and treated, and in how ‘monstrous’ races are all but dismissed as ‘people’ or being part of the world. I can sum a lot of the narrative design and world build up as ‘If they don’t look like us, we either want to have sex with them or kill them.’”`

That discussion on half-species is new to me, but once I read it then I re-read half-elf and half-dwarf, I thought "oh, ya, that's fucked up".

There is no room for what they discuss in our hobby.

<https://medium.com/@arcanistpress/orcs-and-racism-in-d-d-6071a66bf40d> is a good article that talks about this, in addition to the ones I linked on Github
